src/keys/FileKey.cpp

```diff
@@ -402,14 +389,14 @@ bool FileKey::loadHex(QIODevice* device)
     }
 
     QByteArray key = QByteArray::fromHex(data);
-    sodium_memzero(data.data(), static_cast<std::size_t>(data.capacity()));
+    Botan::secure_scrub_memory(data.data(), static_cast<std::size_t>(data.capacity()));
```

Comment:
Here also.

Reply:
It will just be easier to do this when we implement a secure container that bridges between Botan::secure_vector and QByteArray. We use several convenience calls in this class like fromHex, fromBase64, toUpper, toLower, etc. I replaced with secure_vector where I could for now.

Reply:
Sounds good.